test/shared_utils/README.md

```diff
@@ -8,7 +10,19 @@ nsidc-icesat2-image: Dockerfile nsidc-icesat2/environment.yaml
 	--build-arg notebook=NSIDC-ICESAT2_Regression.ipynb --build-arg sub_dir=nsidc-icesat2 --build-arg shared_utils=true .
 ```
 
-Doing this will cause this directory and all its files to be included at `/workdir/shared_utils` in your container.
+Doing this will cause this directory and all its files to be included at `/workdir/shared_utils` in your container when you are working locally.
+
+## Update github workflows to include the build arg for your tests.
+
+To include the shared_utils directory on the regression image built by GitHub you add a `shared_utils` key to the service matrix under your service like was done for the trajectory subsetter in the `.github/workflows/build-all-images.yml` file.
+
+```yml
+          -
+            image: "trajectory-subsetter"
+            notebook: "TrajectorySubsetter_Regression.ipynb"
+            shared-utils: "true"
+
+```
 
 ## Include the necessary python packages in your test's pip_requirements.txt
```

Comment:
Should this say something like, "Include the necessary python pip packages in your test's environment.yaml" since the regression test have environment.yaml's instead of pip_requirements.txt?

Reply:
cb82cba